### DISCUSSION TITLE
Finally getting started on replacing the placeholder icons I made lol

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [mayo78/vsRobloxButNotReally#6] [Made something!]
![59CD62B0-AF80-4E02-B54E-40C670172E12](https://user-images.githubusercontent.com/87618246/160877044-a87e065f-f16d-48c8-8c59-91064b5ca708.png)

**Comment by Theoyeah** [comment]
wait, you dont need i have already done that littelary 25 minutes ago

**Reply by ThatOneSillyGuyDon** [reply]
> wait, you dont need i have already done that littelary 25 minutes ago

Yeah, but I don’t like stealing assets directly from other engines, sooooo

**Reply by Theoyeah** [reply]
but you stealed the old icon too

**Reply by ThatOneSillyGuyDon** [reply]
> but you stealed the old icon too

Old BF, Dad & GF icons were my art, but was outdated and I wanted to remake it, so check your facts before you talk

**Reply by ThatOneSillyGuyDon** [reply]
https://user-images.githubusercontent.com/87618246/160884050-fd5f5879-f739-442e-9218-163287a6dca3.mov

**Reply by Theoyeah** [reply]
OH YOUR ICONS YOU REMADE FOR ME IS THIS HERE !!1!



btw, what the name of your app ?

**Reply by ThatOneSillyGuyDon** [reply]
> OH YOUR ICONS YOU REMADE FOR ME IS THIS HERE !!1!
> 
> btw, what the name of your app ?

Autodesk Sketchbook

I recently learned that for some reason, they made the desktop version cost money now, due to the so called “security reasons”, while the mobile app remains to be free to download

**Reply by Theoyeah** [reply]
Could you just regive me the icons you remade, i wanna try to make the winning icon

**Reply by ThatOneSillyGuyDon** [reply]
> Could you just regive me the icons you remade, i wanna try to make the winning icon

Was it the red BF icon?

**Reply by Theoyeah** [reply]
no, red "from angry birds" and the grey guy (btw it is a sheep)

**Reply by ThatOneSillyGuyDon** [reply]
K

**Reply by ThatOneSillyGuyDon** [reply]
<img width="3000" alt="DC509B10-6AFD-4396-9D94-4E8F3C173E2A" src="https://user-images.githubusercontent.com/87618246/160885760-406286b8-de72-4800-9df5-9906cbf2a154.png">
<img width="3000" alt="F24DA8D9-D639-41D3-AFFD-3FB5730CB624" src="https://user-images.githubusercontent.com/87618246/160885765-6d1fab14-da93-43f3-b4d5-9b00f13510d6.png">

**Reply by ThatOneSillyGuyDon** [reply]
Yeah, there’s an empty space I left for the winning icon

**Reply by Theoyeah** [reply]
thanks (:

**Reply by Theoyeah** [reply]
look at the winning icon i made !
![icon-red](https://user-images.githubusercontent.com/97792861/161001285-7b26c93e-ae19-49d6-a346-e96761a50c19.png)
![redcorrupted](https://user-images.githubusercontent.com/97792861/161001303-e548654c-66ba-4e95-b4ad-c617ca0d1a42.png)
<img width="225" alt="poin" src="https://user-images.githubusercontent.com/97792861/161001350-af361080-4b85-4943-ba04-f3e7bd7c7f59.png">


i used paint.net for windows "free"
